src/Tizen.NUI/src/public/Rendering/Shader.cs

```diff
@@ -118,32 +118,22 @@ protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef
         /// The Hint class is used to provide additional information to the shader.
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
```

Comment:
```suggestion
        [EditorBrowsable(EditorBrowsableState.Never)]
```

Please mark it as hidden.